ui/app/src/components/Navigation/PathNavigator/Widget.tsx

```diff
@@ -14,7 +14,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import React, { Fragment, useCallback, useReducer, useState } from 'react';
+import React, { Fragment, useEffect, useState } from 'react';
```

Comment:
Heads up: merge coming your way due to [this PR](https://github.com/craftercms/studio-ui/pull/1868) getting merged